implementations/cats/src/test/scala/sttp/client/impl/cats/CatsTestBase.scala

```diff
@@ -0,0 +1,14 @@
+package sttp.client.impl.cats
+
+import cats.effect.{Blocker, ContextShift, IO}
+import sttp.client.testing.ConvertToFuture
+
+import scala.concurrent.ExecutionContext
+
+trait CatsTestBase {
```

Comment:
isn't this usable also in other tests? since we already have it :)

Reply:
Do you mean using it in more tests? Or is this duplicated somewhere?

There's many duplicate implementations of `ConvertToFuture` in different places, that I thought I'd take a look at cleaning up.

Reply:
thanks!